src/python/pants/backend/python/subsystems/setup.py

```diff
@@ -344,31 +342,6 @@ class PythonSetup(Subsystem):
     @property
     def generate_lockfiles_with_pex(self) -> bool:
         """Else, generate with Poetry."""
```

Comment:
It's pre-existing but the diff drew my attention to it - strange doc comment since it assumes use of the returned bool in an if / else. That need not be the case. Who knows how the bool will be used. Seems like a doc comment that should be dropped some day when in the neighborhood.

Reply:
Fair. This will be deleted possibly in 2.13 if we deprecate the poetry generator